src/index.ts

```diff
@@ -537,6 +540,14 @@ or make sure that you already have Serverless 3.x installed in your project.
       environment.NEW_RELIC_SERVERLESS_MODE_ENABLED = "true";
     }
 
+    // Uses same layer as CLI so the paths will be the same
```

Comment:
Could you please look into the `lint` issue because of which the checks are failing?

Reply:
Working on it.